### PR TITLE
tests/manual: Update and fix text

### PR DESCRIPTION
**Pull request by tari01**
@sunweaver 

I have modified this the best I could, but I wonder if we need this file at all - it seems to be an orphan...

### CODE REVIEW

File: tests/manual [hunk 1]
```diff
@@ -1,25 +1,25 @@
 
-Test-case ayatana-indicator-datetime/unity7-items-check
+Test-case ayatana-indicator-datetime/MATE-and-other-desktops
 <dl>
```
Comment (sunweaver) [inline tests/manual:3]: mate-items-check?

File: tests/manual [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
-	<dt>Log in to a Unity 7 user session</dt>
-	<dt>Go to the panel and click on the DateTime indicator</dt>
-		<dd>Ensure there are items in the menu</dd>
+    <dt>Log in to a desktop user session</dt>
+    <dt>Go to the panel and click on the DateTime indicator</dt>
+        <dd>Ensure there are items in the menu</dd>
 </dl>
 
-Test-case ayatana-indicator-datetime/unity7-greeter-items-check
+Test-case ayatana-indicator-datetime/MATE-and-other-desktops-greeter-items-check
```
Comment (sunweaver) [inline tests/manual:9]: arctica-greeter-items-check?

File: tests/manual [hunk 3]
```diff
@@ -31,116 +31,116 @@ Test-case ayatana-indicator-datetime/timestamp-wakeup
...
-	<dt>Create and save an upcoming alarm in ubuntu-clock-app</dt>
-	<dt>Unplug the phone from any USB connection and put it to sleep</dt>
-		<dd>Confirm that the alarm sounds on time even if the phone is asleep.  (Note: if in doubt about sleep you can see in the syslog whether the device actually suspended or whether the suspend was aborted)</dd>
-		<dd>Confirm that the screen comes on when the alarm is triggered.<dd>
-		<dd>If the device supports haptic feedback, confirm the alarm vibrates.</dd>
+    <dt>Create and save an upcoming alarm in lomiri-clock-app</dt>
+    <dt>Unplug the phone from any USB connection and put it to sleep</dt>
+        <dd>Confirm that the alarm sounds on time even if the phone is asleep.  (Note: if in doubt about sleep you can see in the syslog whether the device actually suspended or whether the suspend was aborted)</dd>
+        <dd>Confirm that the screen comes on when the alarm is triggered.<dd>
+        <dd>If the device supports haptic feedback, confirm the alarm vibrates.</dd>
```
Comment (sunweaver) [inline tests/manual:38]: possibly add a FIXME here? Haptics are currently commented out / not yet ported to hfd.

**Review by sunweaver — CHANGES_REQUESTED** [review]
Please look at my suggestion for changes and nod / shake head...

**Comment by tari01** [comment]
> Please look at my suggestions for changes and nod / shake head...

Nod. Makes sense.

**Review by sunweaver — APPROVED** [review]
LGTM

**Pull request merged**